tests/testthat/test-correspondenceanalysis.R

```diff
@@ -546,3 +588,78 @@ test_that("Output contains the right class for extension buttons", {
 
     expect_true(inherits(result, "CorrespondenceAnalysis"))
 })
+
+test_that("Bubble sizes + rows removed",
+{
+    x <- structure(c(6.42201834862385, 57.4923547400612, 22.3241590214067,
+        8.56269113149847, 60.5504587155963, 10.0917431192661, 9.78593272171254,
+        100, 1.8348623853211, 58.7155963302752, 55.045871559633, 2.14067278287462,
+        57.7981651376147, 30.8868501529052, 17.4311926605505, 100, 9.1743119266055,
+        22.9357798165138, 12.8440366972477, 9.78593272171254, 43.4250764525994,
+        7.3394495412844, 29.9694189602446, 100, 65.1376146788991, 21.7125382262997,
+        4.58715596330275, 37.9204892966361, 9.1743119266055, 6.42201834862385,
+        8.56269113149847, 100, 22.6299694189602, 9.1743119266055, 51.9877675840979,
+        15.5963302752294, 16.2079510703364, 50.4587155963303, 11.9266055045872,
+        100, 26.2996941896024, 4.58715596330275, 31.4984709480122, 17.737003058104,
+        3.97553516819572, 44.6483180428135, 15.9021406727829, 100, 9.1743119266055,
+        23.5474006116208, 9.1743119266055, 14.3730886850153, 29.6636085626911,
+        6.42201834862385, 38.8379204892966, 100, 92.3547400611621, 14.6788990825688,
+        3.05810397553517, 53.822629969419, 3.36391437308868, 3.97553516819572,
+        2.75229357798165, 100, 0.611620795107034, 76.4525993883792, 64.5259938837921,
+        0, 76.4525993883792, 40.6727828746177, 5.5045871559633, 100,
+        98.1651376146789, 92.3547400611621, 90.8256880733945, 78.8990825688073,
+        95.1070336391437, 86.8501529051988, 57.4923547400612, 100), statistic = "%", .Dim = c(8L,
+        10L), .Dimnames = list(c("Coke", "Diet Coke", "Coke Zero", "Pepsi",
+        "Diet Pepsi", "Pepsi Max", "None of these", "NET"), c("Feminine",
+        "Health-conscious", "Innocent", "Older", "Open to new experiences",
+        "Rebellious", "Sleepy", "Traditional", "Weight-conscious", "NET"
+        )), basedescriptiontext = "sample size = 327", basedescription = list(
+        Minimum = 327L, Maximum = 327L, Range = FALSE, Total = 327L,
+        Missing = 0L, EffectiveSampleSize = 327L, EffectiveSampleSizeProportion = 100,
+        FilteredProportion = 0), questiontypes = "PickAnyGrid", span = list(
+        rows = structure(list(c("Coke", "Diet Coke", "Coke Zero",
+        "Pepsi", "Diet Pepsi", "Pepsi Max", "None of these", "NET"
+        )), class = "data.frame", .Names = "", row.names = c(NA,
+        8L)), columns = structure(list(c("Feminine", "Health-conscious",
+        "Innocent", "Older", "Open to new experiences", "Rebellious",
+        "Sleepy", "Traditional", "Weight-conscious", "NET")), class = "data.frame", .Names = "", row.names = c(NA,
+        10L))), name = "table.Q5", questions = c("Q5", "SUMMARY"))
+
+    b.size <- structure(c(6.42201834862385, 57.4923547400612, 22.3241590214067,
```

Comment:
Is this exactly the same as `x`? If so, why is it defined again?

Reply:
Sorry, this was work in progress that I forgot to remove. I've deleted this section now in 35da5f2